dev/com.ibm.ws.ejbcontainer.timer/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -30,6 +30,14 @@
         <AD name="%persistentExecutorRef" description="%persistentExecutorRef.desc"
             id="persistentExecutorRef" type="String" required="false" ibm:type="pid" ibm:reference="com.ibm.ws.concurrent.persistent.executor"/>
         <AD id="persistentExecutor.target" type="String"  ibm:final="true" default="(service.pid=${persistentExecutorRef})" name="internal" description="internal use only"/>
+        					
```

Comment:
Would be nice if the indent matched the above entries.... probably some tab vs blank problem.  If you just copy the white space from above to your new lines, things should match up.

Reply:
Made change